CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## [1.2.16](https://github.com/ably/ably-java/tree/v1.2.16)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.15...v1.2.16)
+
+In this release, we have fixed a bug that was introduced in 1.2.15 that caused the SDK to return early from the 
+`Auth#renewAuth` method.
+
+- call waiter.close\(\) after breaking from while loop [\#825](https://github.com/ably/ably-java/pull/825) ([ikbalkaya](https://github.com/ikbalkaya))
```

Comment:
Do we need those `\` in this issue name?
```suggestion
- call waiter.close() after breaking from while loop [\#825](https://github.com/ably/ably-java/pull/825) ([ikbalkaya](https://github.com/ikbalkaya))
```

Reply:
Probably not. changelog generator gave this to me so I thought it would be about rendering. Let me apply your suggestion